docs/client/install-kn.md

```diff
@@ -7,18 +7,27 @@ components:
 function: how-to
 ---
 
-# Installing the Knative CLI
+# Installing CLI Tools
 
-This guide provides details about how you can install the Knative `kn` CLI.
+There are three CLI tools available for managing Knative:
+
+- Kubernetes CLI - `kubectl`
+- Knative CLI - `kn`
+- Knative Operator CLI - `kn`
```

Comment:
The operator is part of the Knative CLI, but is primarily focused on administrators who have whole-cluster control.  I think this documentation should be focused on developers, and we should leave out the operator bits.